src/topology/covering.lean

```diff
@@ -0,0 +1,110 @@
+/-
+Copyright (c) 2022 Thomas Browning. All rights reserved.
+Released under Apache 2.0 license as described in the file LICENSE.
+Authors: Thomas Browning
+-/
+import topology.is_locally_homeomorph
+import topology.fiber_bundle
+
+/-!
+# Covering Maps
+
+This file defines covering maps.
+
+## Main definitions
+
+* `is_covering_map`: A covering map is a continuous function `f : E → X` with discrete
+  fibers such that each point of `X` has an evenly covered neighborhood.
```

Comment:
Could you define evenly covered neighborhoods, and also insist on the fact that your definition does *not* imply that the fibers are all isomorphic when the target space is not connected? Since you can find both definitions of covering map in the literature, one should remove any ambiguity here.